1.32.0

matteobruni opened the pull request:
# tsParticles 1.32.0

## Breaking Changes

- Removed some old unused alias types
- Removed the infection plugin from the library, it's an external plugin now
- Removed light interaction from the core library, it's an external plugin now

## Bug Fixes

- Improved particles links, closes #1848 
- Fixed out of canvas out mode

## New Features

- Improved roll back color, now darken and enlighten can be set both and one of them will be picked randomly for each particle and will work also for animated colors
- Added particles orbits external plugin, closes #188
- Added particle repulse external plugin, closes #609

dependabot[bot] committed:
build(deps-dev): bump riot from 5.4.5 to 6.0.0

Bumps [riot](https://github.com/riot/riot) from 5.4.5 to 6.0.0.
- [Release notes](https://github.com/riot/riot/releases)
- [Commits](https://github.com/riot/riot/compare/v5.4.5...v6.0.0)

---
updated-dependencies:
- dependency-name: riot
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump svelte-check from 2.2.1 to 2.2.2

Bumps [svelte-check](https://github.com/sveltejs/language-tools) from 2.2.1 to 2.2.2.
- [Release notes](https://github.com/sveltejs/language-tools/releases)
- [Commits](https://github.com/sveltejs/language-tools/compare/svelte-check-2.2.1...svelte-check-2.2.2)

---
updated-dependencies:
- dependency-name: svelte-check
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @types/node from 16.0.3 to 16.3.1

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 16.0.3 to 16.3.1.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1794 from matteobruni/dependabot/npm_and_yarn/dev/svelte-check-2.2.2

build(deps-dev): bump svelte-check from 2.2.1 to 2.2.2

matteobruni committed:
Merge pull request #1792 from matteobruni/dependabot/npm_and_yarn/dev/riot-6.0.0

build(deps-dev): bump riot from 5.4.5 to 6.0.0

matteobruni committed:
Merge pull request #1795 from matteobruni/dependabot/npm_and_yarn/dev/types/node-16.3.1

build(deps-dev): bump @types/node from 16.0.3 to 16.3.1

dependabot[bot] committed:
build(deps-dev): bump @riotjs/cli from 5.3.0 to 6.0.4

Bumps [@riotjs/cli](https://github.com/riot/cli) from 5.3.0 to 6.0.4.
- [Release notes](https://github.com/riot/cli/releases)
- [Changelog](https://github.com/riot/cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/riot/cli/compare/v5.3.0...v6.0.4)

---
updated-dependencies:
- dependency-name: "@riotjs/cli"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1796 from matteobruni/dependabot/npm_and_yarn/dev/riotjs/cli-6.0.4

build(deps-dev): bump @riotjs/cli from 5.3.0 to 6.0.4

dependabot[bot] committed:
build(deps-dev): bump typedoc from 0.21.2 to 0.21.4

Bumps [typedoc](https://github.com/TypeStrong/TypeDoc) from 0.21.2 to 0.21.4.
- [Release notes](https://github.com/TypeStrong/TypeDoc/releases)
- [Commits](https://github.com/TypeStrong/TypeDoc/compare/v0.21.2...v0.21.4)

---
updated-dependencies:
- dependency-name: typedoc
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @riotjs/compiler from 5.4.2 to 6.0.0

Bumps [@riotjs/compiler](https://github.com/riot/compiler) from 5.4.2 to 6.0.0.
- [Release notes](https://github.com/riot/compiler/releases)
- [Changelog](https://github.com/riot/compiler/blob/main/CHANGELOG.md)
- [Commits](https://github.com/riot/compiler/compare/v5.4.2...v6.0.0)

---
updated-dependencies:
- dependency-name: "@riotjs/compiler"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump babel-preset-solid from 1.0.0 to 1.0.1

Bumps [babel-preset-solid](https://github.com/solidjs/solid) from 1.0.0 to 1.0.1.
- [Release notes](https://github.com/solidjs/solid/releases)
- [Changelog](https://github.com/solidjs/solid/blob/main/CHANGELOG.md)
- [Commits](https://github.com/solidjs/solid/compare/v1.0.0...v1.0.1)

---
updated-dependencies:
- dependency-name: babel-preset-solid
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump ng-packagr from 12.1.0 to 12.1.1

Bumps [ng-packagr](https://github.com/ng-packagr/ng-packagr) from 12.1.0 to 12.1.1.
- [Release notes](https://github.com/ng-packagr/ng-packagr/releases)
- [Changelog](https://github.com/ng-packagr/ng-packagr/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ng-packagr/ng-packagr/compare/v12.1.0...v12.1.1)

---
updated-dependencies:
- dependency-name: ng-packagr
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1793 from matteobruni/dependabot/npm_and_yarn/dev/riotjs/compiler-6.0.0

build(deps-dev): bump @riotjs/compiler from 5.4.2 to 6.0.0

dependabot[bot] committed:
build(deps-dev): bump svelte-preprocess from 4.7.3 to 4.7.4

Bumps [svelte-preprocess](https://github.com/sveltejs/svelte-preprocess) from 4.7.3 to 4.7.4.
- [Release notes](https://github.com/sveltejs/svelte-preprocess/releases)
- [Changelog](https://github.com/sveltejs/svelte-preprocess/blob/main/CHANGELOG.md)
- [Commits](https://github.com/sveltejs/svelte-preprocess/commits)

---
updated-dependencies:
- dependency-name: svelte-preprocess
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump rollup from 2.52.8 to 2.53.1

Bumps [rollup](https://github.com/rollup/rollup) from 2.52.8 to 2.53.1.
- [Release notes](https://github.com/rollup/rollup/releases)
- [Changelog](https://github.com/rollup/rollup/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup/compare/v2.52.8...v2.53.1)

---
updated-dependencies:
- dependency-name: rollup
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1798 from matteobruni/dependabot/npm_and_yarn/dev/typedoc-0.21.4

build(deps-dev): bump typedoc from 0.21.2 to 0.21.4

matteobruni committed:
Merge pull request #1799 from matteobruni/dependabot/npm_and_yarn/dev/babel-preset-solid-1.0.1

build(deps-dev): bump babel-preset-solid from 1.0.0 to 1.0.1

matteobruni committed:
Merge pull request #1800 from matteobruni/dependabot/npm_and_yarn/dev/ng-packagr-12.1.1

build(deps-dev): bump ng-packagr from 12.1.0 to 12.1.1

matteobruni committed:
Merge pull request #1797 from matteobruni/dependabot/npm_and_yarn/dev/rollup-2.53.1

build(deps-dev): bump rollup from 2.52.8 to 2.53.1

matteobruni committed:
Merge pull request #1801 from matteobruni/dependabot/npm_and_yarn/dev/svelte-preprocess-4.7.4

build(deps-dev): bump svelte-preprocess from 4.7.3 to 4.7.4

dependabot[bot] committed:
build(deps-dev): bump ts-node from 10.0.0 to 10.1.0

Bumps [ts-node](https://github.com/TypeStrong/ts-node) from 10.0.0 to 10.1.0.
- [Release notes](https://github.com/TypeStrong/ts-node/releases)
- [Commits](https://github.com/TypeStrong/ts-node/compare/v10.0.0...v10.1.0)

---
updated-dependencies:
- dependency-name: ts-node
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1802 from matteobruni/dependabot/npm_and_yarn/dev/ts-node-10.1.0

build(deps-dev): bump ts-node from 10.0.0 to 10.1.0

matteobruni committed:
chore(engine): preparing the core library to the v2 structure

matteobruni committed:
build: restored riot to v5 for now, it's broken and don't know how to fix that

matteobruni committed:
build: removed some old unused alias types

matteobruni committed:
Merge branch 'staging' into dev

dependabot[bot] committed:
build(deps-dev): bump @typescript-eslint/parser from 4.28.2 to 4.28.3

Bumps [@typescript-eslint/parser](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/parser) from 4.28.2 to 4.28.3.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/parser/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v4.28.3/packages/parser)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/parser"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump riot from 5.4.5 to 6.0.0

Bumps [riot](https://github.com/riot/riot) from 5.4.5 to 6.0.0.
- [Release notes](https://github.com/riot/riot/releases)
- [Commits](https://github.com/riot/riot/compare/v5.4.5...v6.0.0)

---
updated-dependencies:
- dependency-name: riot
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump style-loader from 3.0.0 to 3.1.0

Bumps [style-loader](https://github.com/webpack-contrib/style-loader) from 3.0.0 to 3.1.0.
- [Release notes](https://github.com/webpack-contrib/style-loader/releases)
- [Changelog](https://github.com/webpack-contrib/style-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack-contrib/style-loader/compare/v3.0.0...v3.1.0)

---
updated-dependencies:
- dependency-name: style-loader
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1808 from matteobruni/dependabot/npm_and_yarn/dev/style-loader-3.1.0

build(deps-dev): bump style-loader from 3.0.0 to 3.1.0

matteobruni committed:
Merge pull request #1805 from matteobruni/dependabot/npm_and_yarn/dev/typescript-eslint/parser-4.28.3

build(deps-dev): bump @typescript-eslint/parser from 4.28.2 to 4.28.3

dependabot[bot] committed:
build(deps-dev): bump @typescript-eslint/eslint-plugin

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 4.28.2 to 4.28.3.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v4.28.3/packages/eslint-plugin)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/eslint-plugin"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1810 from matteobruni/dependabot/npm_and_yarn/dev/typescript-eslint/eslint-plugin-4.28.3

build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.28.2 to 4.28.3

matteobruni committed:
Merge pull request #1806 from matteobruni/dependabot/npm_and_yarn/dev/riot-6.0.0

build(deps-dev): bump riot from 5.4.5 to 6.0.0

dependabot[bot] committed:
build(deps-dev): bump @riotjs/cli from 5.3.0 to 6.0.4

Bumps [@riotjs/cli](https://github.com/riot/cli) from 5.3.0 to 6.0.4.
- [Release notes](https://github.com/riot/cli/releases)
- [Changelog](https://github.com/riot/cli/blob/main/CHANGELOG.md)
- [Commits](https://github.com/riot/cli/compare/v5.3.0...v6.0.4)

---
updated-dependencies:
- dependency-name: "@riotjs/cli"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1807 from matteobruni/dependabot/npm_and_yarn/dev/riotjs/cli-6.0.4

build(deps-dev): bump @riotjs/cli from 5.3.0 to 6.0.4

dependabot[bot] committed:
build(deps-dev): bump @riotjs/compiler from 5.4.2 to 6.0.0

Bumps [@riotjs/compiler](https://github.com/riot/compiler) from 5.4.2 to 6.0.0.
- [Release notes](https://github.com/riot/compiler/releases)
- [Changelog](https://github.com/riot/compiler/blob/main/CHANGELOG.md)
- [Commits](https://github.com/riot/compiler/compare/v5.4.2...v6.0.0)

---
updated-dependencies:
- dependency-name: "@riotjs/compiler"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1809 from matteobruni/dependabot/npm_and_yarn/dev/riotjs/compiler-6.0.0

build(deps-dev): bump @riotjs/compiler from 5.4.2 to 6.0.0

matteobruni committed:
build: updated riot v6 again, trying to fix the issue with external support

ghost commented:
## DeepCode's analysis on [#70a566](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/39dd2c3e905fba7240da4714f5efd6dc794b74f1/matteobruni/tsparticles/70a56681250ae40ef2f4e9c86de009ff2331988a/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=2&) found:

- :information_source: **2** minor issues. :point_down:
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">No catch method for promise. This may result in an unhandled promise rejection. Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/70a56681250ae40ef2f4e9c86de009ff2331988a/components/riot/riot-particles.js#L32">riot-particles.js:32</a></li> <li><a href="https://github.com/matteobruni/tsparticles/blob/70a56681250ae40ef2f4e9c86de009ff2331988a/components/riot/riot-particles.js#L34">riot-particles.js:34</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/39dd2c3e905fba7240da4714f5efd6dc794b74f1/matteobruni/tsparticles/70a56681250ae40ef2f4e9c86de009ff2331988a/pr/_/%2Fcomponents%2Friot%2Friot-particles.js/javascript%2Fdc_interfile_project%2FPromiseNotCaughtGeneral/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
    <tr>
      <td width="77%">map does not modify this array, it returns a new object. Ignoring the return value of map is not the intended use of this api. Did you mean to modify this array? Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/70a56681250ae40ef2f4e9c86de009ff2331988a/demo/ionic/src/test.ts#L25">test.ts:25</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/39dd2c3e905fba7240da4714f5efd6dc794b74f1/matteobruni/tsparticles/70a56681250ae40ef2f4e9c86de009ff2331988a/pr/_/%2Fdemo%2Fionic%2Fsrc%2Ftest.ts/javascript%2Fdc_interfile_project%2FPureMethodReturnValueIgnored%2Ftest/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/39dd2c3e905fba7240da4714f5efd6dc794b74f1/matteobruni/tsparticles/70a56681250ae40ef2f4e9c86de009ff2331988a/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=2&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)
#### 👉 The DeepCode service and API will be deprecated in August, 2021. [Here](https://www.deepcode.ai/blog/migrate-to-snyk-code) is the information how to migrate. Thank you for using DeepCode 🙏 ❤️ !
If you are using our plugins, you might be interested in their successors: [Snyk's JetBrains plugin](https://snyk.co/udpkq) and [Snyk's VS Code plugin](https://snyk.co/udpkr).

matteobruni committed:
build: improved confetti back color

matteobruni committed:
build: updated riot demo

dependabot[bot] committed:
build(deps-dev): bump ng-packagr from 12.1.1 to 12.1.2

Bumps [ng-packagr](https://github.com/ng-packagr/ng-packagr) from 12.1.1 to 12.1.2.
- [Release notes](https://github.com/ng-packagr/ng-packagr/releases)
- [Changelog](https://github.com/ng-packagr/ng-packagr/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ng-packagr/ng-packagr/compare/v12.1.1...v12.1.2)

---
updated-dependencies:
- dependency-name: ng-packagr
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @types/node from 16.3.1 to 16.3.2

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 16.3.1 to 16.3.2.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump css-loader from 5.2.6 to 5.2.7

Bumps [css-loader](https://github.com/webpack-contrib/css-loader) from 5.2.6 to 5.2.7.
- [Release notes](https://github.com/webpack-contrib/css-loader/releases)
- [Changelog](https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack-contrib/css-loader/compare/v5.2.6...v5.2.7)

---
updated-dependencies:
- dependency-name: css-loader
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1811 from matteobruni/dependabot/npm_and_yarn/dev/ng-packagr-12.1.2

build(deps-dev): bump ng-packagr from 12.1.1 to 12.1.2

matteobruni committed:
Merge pull request #1812 from matteobruni/dependabot/npm_and_yarn/dev/types/node-16.3.2

build(deps-dev): bump @types/node from 16.3.1 to 16.3.2

matteobruni committed:
Merge pull request #1813 from matteobruni/dependabot/npm_and_yarn/dev/css-loader-5.2.7

build(deps-dev): bump css-loader from 5.2.6 to 5.2.7

matteobruni committed:
build: restored riot v5

matteobruni committed:
Merge remote-tracking branch 'origin/dev' into dev

matteoums committed:
build: improved life updater code

matteobruni committed:
build: prettier

matteobruni committed:
build: updated riot to v6, fixed build

matteobruni committed:
Merge branch 'dev' into riot-v6

# Conflicts:
#	components/riot/package.json

matteobruni committed:
build: merge

matteobruni committed:
build: fixed packages versions

matteobruni committed:
build: try fix broken build on CI

matteobruni committed:
build: updated riot demo yarn.lock file

matteobruni committed:
build: try fix broken build on CI

matteobruni committed:
Merge pull request #1814 from matteobruni/riot-v6

Riot v6 Updated

dependabot[bot] committed:
build(deps): bump @angular/core from 12.1.1 to 12.1.2

Bumps [@angular/core](https://github.com/angular/angular/tree/HEAD/packages/core) from 12.1.1 to 12.1.2.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/12.1.2/packages/core)

---
updated-dependencies:
- dependency-name: "@angular/core"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @angular/cli from 12.1.1 to 12.1.2

Bumps [@angular/cli](https://github.com/angular/angular-cli) from 12.1.1 to 12.1.2.
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Commits](https://github.com/angular/angular-cli/compare/v12.1.1...v12.1.2)

---
updated-dependencies:
- dependency-name: "@angular/cli"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump @angular/platform-browser from 12.1.1 to 12.1.2

Bumps [@angular/platform-browser](https://github.com/angular/angular/tree/HEAD/packages/platform-browser) from 12.1.1 to 12.1.2.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/12.1.2/packages/platform-browser)

---
updated-dependencies:
- dependency-name: "@angular/platform-browser"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump riot from 6.0.0 to 6.0.1

Bumps [riot](https://github.com/riot/riot) from 6.0.0 to 6.0.1.
- [Release notes](https://github.com/riot/riot/releases)
- [Commits](https://github.com/riot/riot/compare/v6.0.0...v6.0.1)

---
updated-dependencies:
- dependency-name: riot
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1822 from matteobruni/dependabot/npm_and_yarn/dev/riot-6.0.1

build(deps-dev): bump riot from 6.0.0 to 6.0.1

matteobruni committed:
Merge pull request #1818 from matteobruni/dependabot/npm_and_yarn/dev/angular/core-12.1.2

build(deps): bump @angular/core from 12.1.1 to 12.1.2

matteobruni committed:
Merge pull request #1819 from matteobruni/dependabot/npm_and_yarn/dev/angular/cli-12.1.2

build(deps-dev): bump @angular/cli from 12.1.1 to 12.1.2

matteobruni committed:
Merge pull request #1821 from matteobruni/dependabot/npm_and_yarn/dev/angular/platform-browser-12.1.2

build(deps): bump @angular/platform-browser from 12.1.1 to 12.1.2

dependabot[bot] committed:
build(deps): bump @angular/animations from 12.1.1 to 12.1.2

Bumps [@angular/animations](https://github.com/angular/angular/tree/HEAD/packages/animations) from 12.1.1 to 12.1.2.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/12.1.2/packages/animations)

---
updated-dependencies:
- dependency-name: "@angular/animations"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump @angular/forms from 12.1.1 to 12.1.2

Bumps [@angular/forms](https://github.com/angular/angular/tree/HEAD/packages/forms) from 12.1.1 to 12.1.2.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/12.1.2/packages/forms)

---
updated-dependencies:
- dependency-name: "@angular/forms"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump @angular/common from 12.1.1 to 12.1.2

Bumps [@angular/common](https://github.com/angular/angular/tree/HEAD/packages/common) from 12.1.1 to 12.1.2.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/12.1.2/packages/common)

---
updated-dependencies:
- dependency-name: "@angular/common"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump css-loader from 5.2.7 to 6.0.0

Bumps [css-loader](https://github.com/webpack-contrib/css-loader) from 5.2.7 to 6.0.0.
- [Release notes](https://github.com/webpack-contrib/css-loader/releases)
- [Changelog](https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack-contrib/css-loader/compare/v5.2.7...v6.0.0)

---
updated-dependencies:
- dependency-name: css-loader
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @angular/compiler-cli from 12.1.1 to 12.1.2

Bumps [@angular/compiler-cli](https://github.com/angular/angular/tree/HEAD/packages/compiler-cli) from 12.1.1 to 12.1.2.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/12.1.2/packages/compiler-cli)

---
updated-dependencies:
- dependency-name: "@angular/compiler-cli"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1823 from matteobruni/dependabot/npm_and_yarn/dev/angular/animations-12.1.2

build(deps): bump @angular/animations from 12.1.1 to 12.1.2

matteobruni committed:
Merge pull request #1824 from matteobruni/dependabot/npm_and_yarn/dev/angular/forms-12.1.2

build(deps): bump @angular/forms from 12.1.1 to 12.1.2

matteobruni committed:
Merge pull request #1825 from matteobruni/dependabot/npm_and_yarn/dev/angular/common-12.1.2

build(deps): bump @angular/common from 12.1.1 to 12.1.2

matteobruni committed:
Merge pull request #1827 from matteobruni/dependabot/npm_and_yarn/dev/angular/compiler-cli-12.1.2

build(deps-dev): bump @angular/compiler-cli from 12.1.1 to 12.1.2

dependabot[bot] committed:
build(deps-dev): bump rollup from 2.53.1 to 2.53.2

Bumps [rollup](https://github.com/rollup/rollup) from 2.53.1 to 2.53.2.
- [Release notes](https://github.com/rollup/rollup/releases)
- [Changelog](https://github.com/rollup/rollup/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rollup/rollup/compare/v2.53.1...v2.53.2)

---
updated-dependencies:
- dependency-name: rollup
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump @angular/platform-browser-dynamic

Bumps [@angular/platform-browser-dynamic](https://github.com/angular/angular/tree/HEAD/packages/platform-browser-dynamic) from 12.1.1 to 12.1.2.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/12.1.2/packages/platform-browser-dynamic)

---
updated-dependencies:
- dependency-name: "@angular/platform-browser-dynamic"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump @angular/router from 12.1.1 to 12.1.2

Bumps [@angular/router](https://github.com/angular/angular/tree/HEAD/packages/router) from 12.1.1 to 12.1.2.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/12.1.2/packages/router)

---
updated-dependencies:
- dependency-name: "@angular/router"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1826 from matteobruni/dependabot/npm_and_yarn/dev/css-loader-6.0.0

build(deps-dev): bump css-loader from 5.2.7 to 6.0.0

matteobruni committed:
Merge pull request #1831 from matteobruni/dependabot/npm_and_yarn/dev/angular/router-12.1.2

build(deps): bump @angular/router from 12.1.1 to 12.1.2

matteobruni committed:
Merge pull request #1830 from matteobruni/dependabot/npm_and_yarn/dev/angular/platform-browser-dynamic-12.1.2

build(deps): bump @angular/platform-browser-dynamic from 12.1.1 to 12.1.2

matteobruni committed:
Merge pull request #1828 from matteobruni/dependabot/npm_and_yarn/dev/rollup-2.53.2

build(deps-dev): bump rollup from 2.53.1 to 2.53.2

dependabot[bot] committed:
build(deps-dev): bump @angular-devkit/build-angular

Bumps [@angular-devkit/build-angular](https://github.com/angular/angular-cli) from 12.1.1 to 12.1.2.
- [Release notes](https://github.com/angular/angular-cli/releases)
- [Commits](https://github.com/angular/angular-cli/compare/v12.1.1...v12.1.2)

---
updated-dependencies:
- dependency-name: "@angular-devkit/build-angular"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1829 from matteobruni/dependabot/npm_and_yarn/dev/angular-devkit/build-angular-12.1.2

build(deps-dev): bump @angular-devkit/build-angular from 12.1.1 to 12.1.2

dependabot[bot] committed:
build(deps): bump @angular/compiler from 12.1.1 to 12.1.2

Bumps [@angular/compiler](https://github.com/angular/angular/tree/HEAD/packages/compiler) from 12.1.1 to 12.1.2.
- [Release notes](https://github.com/angular/angular/releases)
- [Changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md)
- [Commits](https://github.com/angular/angular/commits/12.1.2/packages/compiler)

---
updated-dependencies:
- dependency-name: "@angular/compiler"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @rollup/plugin-typescript from 8.2.1 to 8.2.3

Bumps [@rollup/plugin-typescript](https://github.com/rollup/plugins/tree/HEAD/packages/typescript) from 8.2.1 to 8.2.3.
- [Release notes](https://github.com/rollup/plugins/releases)
- [Changelog](https://github.com/rollup/plugins/blob/master/packages/typescript/CHANGELOG.md)
- [Commits](https://github.com/rollup/plugins/commits/typescript-v8.2.3/packages/typescript)

---
updated-dependencies:
- dependency-name: "@rollup/plugin-typescript"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @rollup/plugin-node-resolve from 13.0.0 to 13.0.2

Bumps [@rollup/plugin-node-resolve](https://github.com/rollup/plugins/tree/HEAD/packages/node-resolve) from 13.0.0 to 13.0.2.
- [Release notes](https://github.com/rollup/plugins/releases)
- [Changelog](https://github.com/rollup/plugins/blob/master/packages/node-resolve/CHANGELOG.md)
- [Commits](https://github.com/rollup/plugins/commits/node-resolve-v13.0.2/packages/node-resolve)

---
updated-dependencies:
- dependency-name: "@rollup/plugin-node-resolve"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @rollup/plugin-commonjs from 19.0.0 to 19.0.1

Bumps [@rollup/plugin-commonjs](https://github.com/rollup/plugins/tree/HEAD/packages/commonjs) from 19.0.0 to 19.0.1.
- [Release notes](https://github.com/rollup/plugins/releases)
- [Changelog](https://github.com/rollup/plugins/blob/master/packages/commonjs/CHANGELOG.md)
- [Commits](https://github.com/rollup/plugins/commits/commonjs-v19.0.1/packages/commonjs)

---
updated-dependencies:
- dependency-name: "@rollup/plugin-commonjs"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1833 from matteobruni/dependabot/npm_and_yarn/dev/angular/compiler-12.1.2

build(deps): bump @angular/compiler from 12.1.1 to 12.1.2

matteobruni committed:
Merge pull request #1834 from matteobruni/dependabot/npm_and_yarn/dev/rollup/plugin-typescript-8.2.3

build(deps-dev): bump @rollup/plugin-typescript from 8.2.1 to 8.2.3

matteobruni committed:
Merge pull request #1835 from matteobruni/dependabot/npm_and_yarn/dev/rollup/plugin-node-resolve-13.0.2

build(deps-dev): bump @rollup/plugin-node-resolve from 13.0.0 to 13.0.2

matteobruni committed:
Merge pull request #1836 from matteobruni/dependabot/npm_and_yarn/dev/rollup/plugin-commonjs-19.0.1

build(deps-dev): bump @rollup/plugin-commonjs from 19.0.0 to 19.0.1

matteobruni committed:
build: improved links to particles

matteobruni committed:
Merge remote-tracking branch 'origin/dev' into dev

matteobruni committed:
build: deps updated

matteoums committed:
build: updating some files from v2

matteobruni committed:
build: fixed merge errors

matteobruni committed:
chore(engine): migrated circle and image shape to new plugin structure

matteobruni committed:
chore(engine): migrated all shapes to new plugin structure and fixed out mode out left and top

matteobruni committed:
chore(engine): migrated some interactors and all updaters to the new plugin system

matteobruni committed:
fix: fixed out of canvas out mode

matteobruni committed:
build: added ionic demo

matteobruni committed:
chore(engine): moved all interactions to the new plugin systems

matteobruni committed:
chore(engine): some refactoring for cleaning some unused code

matteobruni committed:
build: fixed imports in tests

matteobruni committed:
feat: added duration to options

matteobruni committed:
chore(engine): removed the infection plugin from the library, it's an external plugin now

matteobruni committed:
chore(engine): removed light interaction from the core library, it's an external plugin now

matteobruni committed:
build: fixed new readmes

matteobruni committed:
build: fixed editor for infection plugin

dependabot[bot] committed:
build(deps-dev): bump @types/node from 16.3.2 to 16.3.3

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 16.3.2 to 16.3.3.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump eslint-plugin-vue from 7.13.0 to 7.14.0

Bumps [eslint-plugin-vue](https://github.com/vuejs/eslint-plugin-vue) from 7.13.0 to 7.14.0.
- [Release notes](https://github.com/vuejs/eslint-plugin-vue/releases)
- [Commits](https://github.com/vuejs/eslint-plugin-vue/compare/v7.13.0...v7.14.0)

---
updated-dependencies:
- dependency-name: eslint-plugin-vue
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @rollup/plugin-replace from 2.4.2 to 3.0.0

Bumps [@rollup/plugin-replace](https://github.com/rollup/plugins/tree/HEAD/packages/replace) from 2.4.2 to 3.0.0.
- [Release notes](https://github.com/rollup/plugins/releases)
- [Changelog](https://github.com/rollup/plugins/blob/master/packages/replace/CHANGELOG.md)
- [Commits](https://github.com/rollup/plugins/commits/wasm-v3.0.0/packages/replace)

---
updated-dependencies:
- dependency-name: "@rollup/plugin-replace"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1841 from matteobruni/dependabot/npm_and_yarn/dev/types/node-16.3.3

build(deps-dev): bump @types/node from 16.3.2 to 16.3.3

matteobruni committed:
Merge pull request #1845 from matteobruni/dependabot/npm_and_yarn/dev/rollup/plugin-replace-3.0.0

build(deps-dev): bump @rollup/plugin-replace from 2.4.2 to 3.0.0

matteobruni committed:
Merge pull request #1842 from matteobruni/dependabot/npm_and_yarn/dev/eslint-plugin-vue-7.14.0

build(deps-dev): bump eslint-plugin-vue from 7.13.0 to 7.14.0

dependabot[bot] committed:
build(deps-dev): bump css-loader from 6.0.0 to 6.1.0

Bumps [css-loader](https://github.com/webpack-contrib/css-loader) from 6.0.0 to 6.1.0.
- [Release notes](https://github.com/webpack-contrib/css-loader/releases)
- [Changelog](https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack-contrib/css-loader/compare/v6.0.0...v6.1.0)

---
updated-dependencies:
- dependency-name: css-loader
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump svelte-check from 2.2.2 to 2.2.3

Bumps [svelte-check](https://github.com/sveltejs/language-tools) from 2.2.2 to 2.2.3.
- [Release notes](https://github.com/sveltejs/language-tools/releases)
- [Commits](https://github.com/sveltejs/language-tools/compare/svelte-check-2.2.2...svelte-check-2.2.3)

---
updated-dependencies:
- dependency-name: svelte-check
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1846 from matteobruni/dependabot/npm_and_yarn/dev/css-loader-6.1.0

build(deps-dev): bump css-loader from 6.0.0 to 6.1.0

matteobruni committed:
Merge pull request #1847 from matteobruni/dependabot/npm_and_yarn/dev/svelte-check-2.2.3

build(deps-dev): bump svelte-check from 2.2.2 to 2.2.3

matteobruni committed:
build: deps updated

matteobruni committed:
build: deps updated

matteobruni committed:
build: updated firebase demo

matteobruni committed:
feat: added orbits as a plugin from v2, closes #188
feat: added particle repulse as a plugin from v2, closes #609

matteobruni committed:
build: slim build now faster, fix a code style

matteobruni committed:
chore(docs): updating some docs before 1.32 release

matteobruni committed:
chore(docs): updated docs for 1.32.0

dependabot[bot] committed:
build(deps-dev): bump css-loader from 6.1.0 to 6.2.0

Bumps [css-loader](https://github.com/webpack-contrib/css-loader) from 6.1.0 to 6.2.0.
- [Release notes](https://github.com/webpack-contrib/css-loader/releases)
- [Changelog](https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack-contrib/css-loader/compare/v6.1.0...v6.2.0)

---
updated-dependencies:
- dependency-name: css-loader
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1849 from matteobruni/dependabot/npm_and_yarn/dev/css-loader-6.2.0

build(deps-dev): bump css-loader from 6.1.0 to 6.2.0

matteobruni committed:
build: fixed ionic demo lockfile

matteobruni committed:
chore(release): published new version

 - ng-particles@2.15.0
 - inferno-particles@1.15.0
 - jquery-particles@1.32.0
 - preact-particles@1.32.0
 - react-tsparticles@1.32.0
 - riot-particles@1.6.0
 - solid-particles@1.15.0
 - svelte-particles@1.15.0
 - particles.vue@2.15.0
 - particles.vue3@1.15.0
 - web-particles@1.5.0
 - ng-particles-demo@1.17.0
 - editor-demo@1.17.0
 - imports-ts-latest@1.3.0
 - imports-ts34@1.3.0
 - imports-ts38@1.3.0
 - imports-ts40@1.3.0
 - inferno-particles-demo@1.15.0
 - ionic@0.1.0
 - jquery-particles-demo@1.17.0
 - preact-particles-demo@1.17.0
 - react-tsparticles-demo@1.17.0
 - riot-particles-demo@1.6.0
 - solid-particles-demo@1.15.0
 - svelte-demo@1.15.0
 - tsparticles-demo@1.17.0
 - tsparticles-demo-new@1.17.0
 - particles.vue-demo@1.17.0
 - particles.vue3-demo@1.15.0
 - web-particles-demo@1.17.0
 - tsparticles-editor@1.32.0
 - tsparticles@1.32.0
 - tsparticles-interaction-light@1.32.0
 - tsparticles-interaction-particles-repulse@1.32.0
 - tsparticles-plugin-infection@1.32.0
 - tsparticles-preset-big-circles@1.17.0
 - tsparticles-preset-bubbles@1.17.0
 - tsparticles-preset-confetti@1.17.0
 - tsparticles-preset-fire@1.17.0
 - tsparticles-preset-firefly@1.17.0
 - tsparticles-preset-fireworks@1.17.0
 - tsparticles-preset-fountain@1.17.0
 - tsparticles-preset-links@1.17.0
 - tsparticles-preset-sea-anemone@1.17.0
 - tsparticles-preset-snow@1.17.0
 - tsparticles-preset-stars@1.17.0
 - tsparticles-shape-bubble@1.17.0
 - tsparticles-shape-heart@1.17.0
 - tsparticles-shape-multiline-text@1.17.0
 - tsparticles-shape-rounded-rect@1.17.0
 - tsparticles-shape-spiral@1.17.0
 - cra-template-particles-typescript@1.15.0
 - cra-template-particles@1.15.0
 - tsparticles-updater-orbit@1.32.0
 - tsparticles-website@1.32.0

dependabot[bot] committed:
build(deps-dev): bump webpack from 5.44.0 to 5.45.1

Bumps [webpack](https://github.com/webpack/webpack) from 5.44.0 to 5.45.1.
- [Release notes](https://github.com/webpack/webpack/releases)
- [Commits](https://github.com/webpack/webpack/compare/v5.44.0...v5.45.1)

---
updated-dependencies:
- dependency-name: webpack
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @types/node from 16.3.3 to 16.4.0

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 16.3.3 to 16.4.0.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump eslint from 7.30.0 to 7.31.0

Bumps [eslint](https://github.com/eslint/eslint) from 7.30.0 to 7.31.0.
- [Release notes](https://github.com/eslint/eslint/releases)
- [Changelog](https://github.com/eslint/eslint/blob/master/CHANGELOG.md)
- [Commits](https://github.com/eslint/eslint/compare/v7.30.0...v7.31.0)

---
updated-dependencies:
- dependency-name: eslint
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1850 from matteobruni/dependabot/npm_and_yarn/dev/webpack-5.45.1

build(deps-dev): bump webpack from 5.44.0 to 5.45.1

matteobruni committed:
Merge pull request #1851 from matteobruni/dependabot/npm_and_yarn/dev/types/node-16.4.0

build(deps-dev): bump @types/node from 16.3.3 to 16.4.0

matteobruni committed:
Merge pull request #1852 from matteobruni/dependabot/npm_and_yarn/dev/eslint-7.31.0

build(deps-dev): bump eslint from 7.30.0 to 7.31.0

dependabot[bot] committed:
build(deps-dev): bump style-loader from 3.1.0 to 3.2.1

Bumps [style-loader](https://github.com/webpack-contrib/style-loader) from 3.1.0 to 3.2.1.
- [Release notes](https://github.com/webpack-contrib/style-loader/releases)
- [Changelog](https://github.com/webpack-contrib/style-loader/blob/master/CHANGELOG.md)
- [Commits](https://github.com/webpack-contrib/style-loader/compare/v3.1.0...v3.2.1)

---
updated-dependencies:
- dependency-name: style-loader
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @babel/preset-env from 7.14.7 to 7.14.8

Bumps [@babel/preset-env](https://github.com/babel/babel/tree/HEAD/packages/babel-preset-env) from 7.14.7 to 7.14.8.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.14.8/packages/babel-preset-env)

---
updated-dependencies:
- dependency-name: "@babel/preset-env"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @typescript-eslint/parser from 4.28.3 to 4.28.4

Bumps [@typescript-eslint/parser](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/parser) from 4.28.3 to 4.28.4.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/parser/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v4.28.4/packages/parser)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/parser"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps-dev): bump @typescript-eslint/eslint-plugin

Bumps [@typescript-eslint/eslint-plugin](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/eslint-plugin) from 4.28.3 to 4.28.4.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v4.28.4/packages/eslint-plugin)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/eslint-plugin"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
Merge pull request #1854 from matteobruni/dependabot/npm_and_yarn/dev/typescript-eslint/parser-4.28.4

build(deps-dev): bump @typescript-eslint/parser from 4.28.3 to 4.28.4

matteobruni committed:
Merge pull request #1856 from matteobruni/dependabot/npm_and_yarn/dev/style-loader-3.2.1

build(deps-dev): bump style-loader from 3.1.0 to 3.2.1

matteobruni committed:
Merge pull request #1857 from matteobruni/dependabot/npm_and_yarn/dev/babel/preset-env-7.14.8

build(deps-dev): bump @babel/preset-env from 7.14.7 to 7.14.8

matteobruni committed:
Merge pull request #1853 from matteobruni/dependabot/npm_and_yarn/dev/typescript-eslint/eslint-plugin-4.28.4

build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.28.3 to 4.28.4

dependabot[bot] committed:
build(deps-dev): bump @babel/core from 7.14.6 to 7.14.8

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.14.6 to 7.14.8.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.14.8/packages/babel-core)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

matteobruni committed:
build: updated website

matteobruni committed:
Merge remote-tracking branch 'origin/dev' into dev

matteobruni committed:
Merge pull request #1855 from matteobruni/dependabot/npm_and_yarn/dev/babel/core-7.14.8

build(deps-dev): bump @babel/core from 7.14.6 to 7.14.8
